packages/create-experience-builder/package.json

```diff
@@ -25,12 +25,15 @@
   "scripts": {
     "dev": "echo noop",
     "_prebuild": "npm uninstall -g && rimraf dist",
-    "_build": "tsc",
+    "prebuild": "rimraf dist",
+    "build": "tsc",
     "lint": "eslint src --ext '.ts,.tsx,.js,.jsx' --max-warnings 0 --ignore-path ../../.eslintignore",
     "lint:fix": "eslint src --ext '.ts,.tsx,.js,.jsx' --fix",
     "postbuild": "npm install -g",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "depcruise": "depcruise src"
+    "depcruise": "depcruise src",
+    "_postbuild": "npm install -g",
```

Comment:
I'm not sure, again this was part of the 4 original commits that I inherited when branching off of `cli-updates`.  @elylucas might have a better idea?

Reply:
This was to install the cli tool locally in npm cache so you could run `npm init experience-builder` using the local code. I found it easier to just `node ./dist/index.js`, so I renamed the script so it wouldn't run anymore on build. It can be removed, as well as the `postbuild` script above it.

Reply:
- [ ] remove `postbuild`
- [ ] remove `_postbuild`